EasyPost/Services/CarrierAccountService.cs

```diff
@@ -58,5 +66,25 @@ public async Task<CarrierAccount> Retrieve(string id)
         }
 
         #endregion
+
+        private static string GenerateEndpoint(string carrierAccountType)
+        {
+            var registerEndpointCarriers = new List<string>
+            {
+                "UpsAccount",
+                "FedexAccount"
```

Comment:
Let's sort this list. It may grow in the future.

Reply:
Resolved